keyboards/projectcain/vault45/info.json

```
@@ -1,60 +1,24 @@
 {
-  "keyboard_name": "vault 45",
-  "url": "",
-  "maintainer": "projectcain",
-  "width": 13,
-  "height": 4,
-  "layouts": {
-    "LAYOUT": {
-      "layout": [
-        { "label": "Tab", "x": 0, "y": 0 },
-        { "label": "Q", "x": 1, "y": 0 },
-        { "label": "W", "x": 2, "y": 0 },
-        { "label": "E", "x": 3, "y": 0 },
-        { "label": "R", "x": 4, "y": 0 },
-        { "label": "T", "x": 5, "y": 0 },
-        { "label": "Y", "x": 6, "y": 0 },
-        { "label": "U", "x": 7, "y": 0 },
-        { "label": "I", "x": 8, "y": 0 },
-        { "label": "O", "x": 9, "y": 0 },
-        { "label": "P", "x": 10, "y": 0 },
-        { "label": "Back<br>Space", "x": 11, "y": 0 },
-        { "x": 12, "y": 0 },
-        { "label": "Caps", "x": 0, "y": 1, "w": 1.25 },
-        { "label": "A", "x": 1.25, "y": 1 },
-        { "label": "S", "x": 2.25, "y": 1 },
-        { "label": "D", "x": 3.25, "y": 1 },
-        { "label": "F", "x": 4.25, "y": 1 },
-        { "label": "G", "x": 5.25, "y": 1 },
-        { "label": "H", "x": 6.25, "y": 1 },
-        { "label": "J", "x": 7.25, "y": 1 },
-        { "label": "K", "x": 8.25, "y": 1 },
-        { "label": "L", "x": 9.25, "y": 1 },
-        { "label": ":", "x": 10.25, "y": 1 },
-        { "label": "Enter", "x": 11.25, "y": 1, "w": 1.75 },
-        { "label": "Shift", "x": 0, "y": 2, "w": 1.75 },
-        { "label": "Z", "x": 1.75, "y": 2 },
-        { "label": "X", "x": 2.75, "y": 2 },
-        { "label": "C", "x": 3.75, "y": 2 },
-        { "label": "V", "x": 4.75, "y": 2 },
-        { "label": "B", "x": 5.75, "y": 2 },
-        { "label": "N", "x": 6.75, "y": 2 },
-        { "label": "M", "x": 7.75, "y": 2 },
-        { "label": "<", "x": 8.75, "y": 2 },
-        { "label": ">", "x": 9.75, "y": 2 },
-        { "label": "?", "x": 10.75, "y": 2 },
-        { "label": "Shift", "x": 11.75, "y": 2, "w": 1.25 },
-        { "label": "Hyper", "x": 0, "y": 3, "w": 1.25 },
-        { "label": "Super", "x": 1.25, "y": 3 },
-        { "label": "Meta", "x": 2.25, "y": 3, "w": 1.25 },
-        { "label": "Alt", "x": 3.5, "y": 3, "w": 1.25 },
-        { "x": 4.75, "y": 3, "w": 1.75 },
-        { "x": 6.5, "y": 3, "w": 1.75 },
-        { "label": "Meta", "x": 8.25, "y": 3, "w": 1.25 },
-        { "label": "Super", "x": 9.5, "y": 3, "w": 1.25 },
-        { "label": "Fn2", "x": 10.75, "y": 3 },
-        { "label": "Ctrl", "x": 11.75, "y": 3, "w": 1.25 }
-      ]
+    "keyboard_name": "Vault 45", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "width": 13, 
+    "height": 4, 
+    "layouts": {
```

Here's where we alias the old macro to the new `LAYOUT_all`.

```suggestion
    "layout_aliases": {
        "LAYOUT": "LAYOUT_all"
    },
    "layouts": {
```